crates/solidity/outputs/npm/crate/build.rs

```diff
@@ -3,7 +3,7 @@ use infra_utils::{cargo::CargoWorkspace, commands::Command};
 
 fn main() -> Result<()> {
     // Reference here to make sure we recompile when that changes:
-    use solidity_npm_build as _;
+    // use solidity_npm_build as _;
```

Comment:
This is producing a warning now from cargo now that `lib.rs` is removed, and it is no longer a valid target. I suggest either undoing the removal (I removed it in #566) or also removing the `solidity_npm_build` dependency from `solidity_npm_crate`.

Reply:
I took the second option.